### PR TITLE
Fix Cards Emphasis Levels Display

### PR DESCRIPTION
**Pull request by razwan**
**Description**
Refactored the Media Card block and renamed some CSS custom properties to be able to reuse the Block and Content emphasis level controls from that block. Added transitions and checked the display of the Cards Collection block with all emphasis level combinations on both frontend and editor.

**Types of changes**
What types of changes does your code introduce?
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by madalingorbanescu** [comment]
MediaCard

**Comment by madalingorbanescu** [comment]
cardcollection

**Comment by madalingorbanescu** [comment]
Also, this is how the card looks with Contet Area highlighted:
card_highlighted

I think we should invert the colors for the button.

File: src/blocks/cards-collection/init.php [hunk 1]
```diff
@@ -43,10 +53,14 @@ function novablocks_render_cards_collection_block( $attributes, $content ) {
...
 
 		<div class="<?php echo $className; ?>" style="<?php echo $style; ?>">
-			<?php echo '<' . $titleTag . '>' . $attributes['title'] . '</' . $titleTag . '>'; ?>
-			<p class="intro"><?php echo $attributes['subtitle']; ?></p>
-			<div class="novablocks-cards-collection__layout alignfull">
-				<?php echo $content; ?>
+			<div class="wp-block-group__inner-container">
+				<?php echo '<' . $titleTag . ' class="alignwide">' . $attributes['title'] . '</' . $titleTag . '>'; ?>
+				<p class="intro alignwide"><?php echo $attributes['subtitle']; ?></p>
+				<div class="wp-block alignwide">
```
Comment (madalingorbanescu) [inline src/blocks/cards-collection/init.php]: Why did you change from alignfull, to alignwide?

Comment (razwan) [reply src/blocks/cards-collection/init.php]: Change alignment to "wide" for the block contents to be aligned to the Media Card contents for example. We may add the alignment toolbar in the future if full-aligned is an option.

**Comment by razwan** [comment]
Hey @madalingorbanescu made some of the required changes. Also, some adjustments to styling are present on the `novablocks-cards` on the **Rosa2** repo.

>Also, this is how the card looks with Contet Area highlighted:
>card_highlighted
>
>I think we should invert the colors for the button.

This is probably an issue of **Rosa2**, as the problem is also present on the **Media Card** block with the same settings.
